rest/app.js

```diff
@@ -1,16 +1,63 @@
-//DOM MANIPULATION FUNCTIONS
+const insertUsersInTable = (users) => {
+  let dataHTML = getDataHtmlHeader();
+  let searchedData = getDataHtmlHeader();
+
+  document.querySelector("#searchInput").addEventListener("input", (e) => {
+    let searchValue = e.target.value;
+
+    if (searchValue.length >= 3) {
+      const result = users.filter(
```

Comment:
move to a separate function

Reply:
optimisation, analyse  and check if works the same

users.filter(({ name, id, lastName }) =>
                [name, id, lastName].includes(searchValue)
            );